database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -29,6 +29,6 @@ public function up()
      */
     public function down()
     {
-        Schema::drop('users');
+        Schema::drop('channels');
```

Comment:
I was having issues with the migration files. It was later I realise that the migration is done according to the time the migration file was created. I saw the initial error but I thought I have corrected it.